File: android/src/main/kotlin/com/example/design_engine/layer2_core/dark/EEronprimarycontainerDark.kt
package com.example.design_engine.layer2_core.dark
import com.example.design_engine.R
import com.example.design_engine.layer3_logic.AppTheme
object EEronprimarycontainerDark {
    fun getResId(activeTheme: AppTheme): Int {
        return when (activeTheme) {
            AppTheme.SYSTEM -> R.string.csyd19
            AppTheme.STANDARD -> R.color.cstd19
            AppTheme.BLUE -> R.color.cbd19
            AppTheme.GREEN -> R.color.cgd19
            AppTheme.RED -> R.color.crd19
            AppTheme.NOTHING -> R.color.cnod50
        }
    }
}

File: lib/layer2_core/dark/e_eronprimarycontainer_dark.dart
import '../../layer1_raw/design_token.dart';
import '../../layer1_raw/dark/system.dart';
import '../../layer1_raw/dark/standard.dart';
import '../../layer1_raw/dark/blue.dart';
import '../../layer1_raw/dark/green.dart';
import '../../layer1_raw/dark/red.dart';
import '../../layer1_raw/dark/nothing.dart';

class EEronprimarycontainerDark {
  static DesignToken getToken(dynamic activeTheme) {
    switch (activeTheme.toString().split('.').last) {
      case 'system': return csyd19;
      case 'standard': return cstd19;
      case 'blue': return cbd19;
      case 'green': return cgd19;
      case 'red': return crd19;
      case 'nothing': return cnod50;
      default: return cstd19;
    }
  }
}
